grpc/src/test/java/com/linecorp/armeria/client/grpc/GrpcServiceImplErrorFromClientViewTest.java

```diff
@@ -0,0 +1,325 @@
+                assertThat(Status.trailersFromThrowable(cause)).isNotEqualTo(corruptedMetadata);
+            });
+
+            final RequestLog log = captor.get().log().whenComplete().join();
+            assertThat(log.responseStatus()).isEqualTo(HttpStatus.INTERNAL_SERVER_ERROR);
+            assertThat(log.responseCause())
+                    .satisfies(cause -> {
+                        assertThat(cause).isInstanceOf(StatusRuntimeException.class);
+                        assertThat(Status.fromThrowable(cause).getCode()).isEqualTo(
+                                Status.UNKNOWN.getCode());
+                        assertThat(Status.trailersFromThrowable(cause)).satisfies(metadata -> {
+                            assertThat(isSameSizeAndNameAndValues(metadata, corruptedMetadata)).isFalse();
+                        });
+                    });
+        }
+    }
+
+    // Error inside #onError at server streaming RPC
+    @Test
+    void clientUnaryCallServerStreamingOutputCall() throws InterruptedException {
+        try (ClientRequestContextCaptor captor = Clients.newContextCaptor()) {
+            assertThatThrownBy(() -> {
+                final TestServiceBlockingStub client = GrpcClients.newClient(
+                                                                          "http://127.0.0.1:" + serverWithBlockingExecutor.httpPort(),
+                                                                          TestServiceBlockingStub.class)
+                                                                  .withDeadlineAfter(60, TimeUnit.SECONDS);
+                final Iterator<StreamingOutputCallResponse> it = client.streamingOutputCall(
+                        STREAMING_OUTPUT_CALL_REQUEST_MESSAGE);
+                while (it.hasNext()) {
+                    it.next();
+                }
+            }).satisfies(cause -> {
+                assertThat(Status.fromThrowable(cause).getCode()).isEqualTo(Status.CANCELLED.getCode());
+                assertThat(Status.trailersFromThrowable(cause)).isNotEqualTo(corruptedMetadata);
+            });
+
+            final RequestLog log = captor.get().log().whenComplete().join();
+            assertThat(log.responseStatus()).isEqualTo(HttpStatus.OK);
+            assertThat(log.responseCause())
+                    .isInstanceOf(StatusRuntimeException.class)
+                    .satisfies(cause -> {
+                        assertThat(Status.fromThrowable(cause).getCode()).isEqualTo(
+                                Status.CANCELLED.getCode());
+                        assertThat(Status.trailersFromThrowable(cause)).satisfies(metadata -> {
+                            assertThat(isSameSizeAndNameAndValues(metadata, corruptedMetadata)).isFalse();
+                        });
+                    });
+        }
+    }
+
+    // Error inside #onError at bidirectional streaming RPC
+    @Test
+    void fullDuplexCall() throws Exception {
```

Comment:
What do you think of removing the duplex call since it doesn't seem to be adding any value at the moment? (the streaming case is covered by `clientUnaryCallServerStreamingOutputCall` test

Reply:
I see,
 including https://github.com/line/armeria/pull/4686#discussion_r1126101270,
if there's some prior knowledge about Armeria Server, this test may be useless.

My original intention of creating this test is something like as black box testing.
(Just test all scenarios of GRPC.)

Reply:
> What do you think of removing the duplex

Yes. I've removed now.